### ISSUE TITLE
Duplicate nested layout on homepage

### ISSUE THREAD

**Issue by andrixnet** [opencaching/opencaching-pl#1370]
![ocro](https://user-images.githubusercontent.com/3382777/35283316-9958fe9c-0060-11e8-810a-b35d02723e4b.png)

When not logged in, OCRO shows a duplication of the layout, nested below "upcoming events"

**Comment by kojoty** [comment]
@andrixnet, I observed this yesterday night but this effect disappeared after APC cache flush.
Please restart your Apache service. I'll integrate script for cache management today evening tool and write you more.

**Comment by harrieklomp** [comment]
This is also on OCUK

**Comment by andrixnet** [comment]
Indeed, a full stop and then start of apache solved the problem, at this time.

**Comment by kojoty** [comment]
@andrixnet,
> Indeed, a full stop and then start of apache solved the problem, at this time.

Yes, because it is connected with improper value in APCu cache - restart of apache flush the whole cache content.

**Comment by kojoty** [comment]
@deg-pl, please restart apache service at oc.uk

**Comment by andrixnet** [comment]
the problem has reappeared at OCRO

**Comment by andrixnet** [comment]
Problem persists on OCRO even after login, using SeaMonkey

**Comment by andrixnet** [comment]
"state of the game" boxes scroll really fast in the duplicate part.

**Comment by kojoty** [comment]
OK, now I disabled this "effect" :) and I'm working on permanent fix.

**Comment by kojoty** [comment]
This should be fixed now - please reopen if necessary.

**Comment by kojoty** [comment]
If anyone wants to know why this funny bug appears I can explain:
- startPage has RSS feeds list at the bottom of page
- feeds data is stored in apc cache and refreshed once per hour - on the next request after timeout
- because rss feeds data is (or at least can be) downloaded from external servers this process can takes long time so I decided to do it by async ajax request - this request is run only if there is no feeds data in APC cache (if data is retrieved from local memory cache it is loaded to start page directly during the page creation)
- this ajax call download just html which store in the right place at startPage

in short words because of my fault in special conditions there was improper ajax url (set to null) and in a result this ajax call just get the start page html and display it in the place of feeds, but this html contains script which do the same, and the same - so there was infinite loop of request to server :)
